### PR TITLE
Add Split and realdiv op support

### PR DESCRIPTION
**Pull request by Rasterer**
* Add Split and realdiv op support

* Fix the pad calculation in the case of dilated convolution

@srkreddy1238 @nishi-t please help review, thanks

### CODE REVIEW

**Comment by alexeyr** [comment]
For split, there are already 2 pull requests which cover `SplitV` (with list of split sizes passed) as well: #2001 and #2105.

**Comment by yzhliu** [comment]
Thanks for contributing, please request review from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers)

File: nnvm/python/nnvm/frontend/tensorflow.py [hunk 1]
```diff
@@ -252,8 +252,12 @@ def _impl(inputs, attr, params):
                 in_h = input_shape[2]
                 in_w = input_shape[3]
 
-            pad_v = _get_pad_pair(in_h, kernel_h, stride_h)
-            pad_h = _get_pad_pair(in_w, kernel_w, stride_w)
+            dilation_h = attr['dilations'][0]
```
Comment (srkreddy1238) [inline nnvm/python/nnvm/frontend/tensorflow.py]: @Rasterer Can you explain this change ?

Comment (Rasterer) [reply nnvm/python/nnvm/frontend/tensorflow.py]: Please refer to [https://github.com/dmlc/tvm/blob/master/nnvm/src/top/nn/convolution.cc#L100](url) for the affect of dilation on output size. Dilated conv always can be simulated by a normal conv with dilated kernel. So I use dilated kernel size for pad calculation instead.

Comment (Rasterer) [reply nnvm/python/nnvm/frontend/tensorflow.py]: @srkreddy1238 any more comment about this?

**Comment by yzhliu** [comment]
@srkreddy1238 could you verify the changes?

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 2]
```diff
@@ -502,6 +502,43 @@ def test_forward_gather():
     _test_gather((4,3,5,6), (1,4), [[2,1,0,0]], 0, 'float32')
 
 
+#######################################################################
+# Split
+# ------
```
Comment (srkreddy1238) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Under line up to the string pls.

File: nnvm/tests/python/frontend/tensorflow/test_forward.py [hunk 3]
```diff
@@ -502,6 +502,43 @@ def test_forward_gather():
...
+
+def _test_split(in_shape, axis, num_split, dtype):
```
Comment (srkreddy1238) [inline nnvm/tests/python/frontend/tensorflow/test_forward.py]: Can you add another test case with split followed by concat ?

File: nnvm/tests/python/frontend/tensorflow/test_forward.py (hunk 2 above)
Comment (Rasterer) [reply nnvm/tests/python/frontend/tensorflow/test_forward.py]: @srkreddy1238 updated

File: nnvm/tests/python/frontend/tensorflow/test_forward.py (hunk 3 above)
Comment (Rasterer) [reply nnvm/tests/python/frontend/tensorflow/test_forward.py]: @srkreddy1238 updated

**Comment by srkreddy1238** [comment]
cc @yzhliu

**Comment by yzhliu** [comment]
Thanks @Rasterer @srkreddy1238 . This is now merged.

**Pull request merged**